NtupleTools/test/make_ntuples_cfg.py

```diff
@@ -391,6 +391,21 @@
 from FinalStateAnalysis.NtupleTools.customization_jets import preJets
 fs_daughter_inputs['jets'] = preJets(process,options.use25ns,fs_daughter_inputs['jets'],fs_daughter_inputs['vertices'])
 
+########################
+###   jet cleaning   ###
+########################
+process.miniAODJetCleaningEmbedding = cms.EDProducer(
+    "MiniAODJetCleaningEmbedder",
+    jetSrc = cms.InputTag(fs_daughter_inputs['jets']),
+    muSrc = cms.InputTag(fs_daughter_inputs['muons']),
+    eSrc = cms.InputTag(fs_daughter_inputs['electrons'])
```

Comment:
You need to add an electronID and muonID string input here pulled from this parameters file:
https://github.com/uwcms/FinalStateAnalysis/blob/miniAOD_dev_74X/NtupleTools/python/parameters/default.py#L36

```
eID = cms.untracked.string(parameters['j']['e']['selection'],
eDR = cms.untracked.double(parameters['j']['e']['deltaR'],
mID = cms.untracked.string(parameters['j']['m']['selection'],
mDR = cms.untracked.double(parameters['j']['m']['deltaR'],
jID = cms.untracked.string(parameters['j']['selection'],
```